warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__no_stale_vehicle_positions.sql

```diff
@@ -0,0 +1,62 @@
+{{ config(
+    materialized='incremental',
+    incremental_strategy='insert_overwrite',
+    partition_by = {
+        'field': 'date',
+        'data_type': 'date',
+        'granularity': 'day',
+    },
+) }}
+
+{% if is_incremental() %}
+    {% set timestamps = dbt_utils.get_column_values(table=this, column='dt', order_by = 'dt DESC', max_records = 1) %}
+    {% set max_ts = timestamps[0] %}
+{% endif %}
+
```

Comment:
Is it safe to say that everything above this is standard syntax for any incremental table? ie i shouldn't change any of it.

Reply:
For our specific setup yes; this is a hacky way to get hard-coded partition filters into the SQL at runtime.

This isn't necessary if you are not querying views on top of hive-partitioned external tables. See https://github.com/cal-itp/data-infra/blob/main/warehouse/models/staging/audit/stg_audit__cloudaudit_googleapis_com_data_access.sql as another example; similar use of `is_incremental` but no need for the `get_volumn_values()` hackiness.